src/nemo-connectivity/connectionhelper.h

```diff
@@ -57,7 +57,7 @@ class NEMO_CONNECTIVITY_EXPORT ConnectionHelper : public QObject
 {
     Q_OBJECT
     Q_PROPERTY(bool online READ online NOTIFY onlineChanged)
-    Q_PROPERTY(bool selectorVisible READ selectorVisible NOTIFY selectorVisibleChanged)
+    Q_PROPERTY(bool selectorVisible READ selectorVisible WRITE setSelectorVisible NOTIFY selectorVisibleChanged)
```

Comment:
How's setting `selectorVisible` property supposed to actually show/hide the selector? If it's not (and it looks like it's not) then making this property settable would only allow to set it to the value which doesn't match the reality. What's the point of that?

Reply:
fixed